src/test_l1_hdmi_cec_driver.c

```diff
@@ -79,11 +79,9 @@
 */
 #define DEFAULT_LOGICAL_ADDRESS_STB 3
 
-
 /// Set the CEC sink (Display device) logical address here
 #define DEFAULT_LOGICAL_ADDRESS_PANEL 0
 
-
 static int gTestGroup = 1;
 static int gTestID = 1;
 static bool extendedEnumsSupported=false;
```

Comment:
In case of HdmiCecAddLogicalAddress we need to write a +ve test case 
for source device we must return HDMI_CEC_IO_INVALID_ARGUMENT@n@n
https://github.com/rdkcentral/rdk-halif-hdmi_cec/blob/main/include/hdmi_cec_driver.h#L175
check HdmiCecRemoveLogicalAddress()
please check other API's also

Reply:
updated addLogicalAddress and removeLogicalAddress.
Could not find further api which require test modification.

Reply:
can we check the HdmiCecGetLogicalAddress

in this test case test_hdmicec_hal_l1_getLogicalAddress_sinkDevice_positive
we should be test alone only HdmiCecGetLogicalAddress() except @pre condition API's

https://github.com/rdkcentral/rdk-halif-hdmi_cec/blob/main/include/hdmi_cec_driver.h#L233

Reply:
Removed the additional api calls from test_hdmicec_hal_l1_getLogicalAddress_sinkDevice_positive,
Similar situation were found in HdmiCecTx, But discussion with @srinivasgtl , we came to a conclusion that for those API we need to update interface file to update @post and @pre conditions.

Reply:
do we required separate function for HdmiCecGetLogicalAddress() 
if not required please remove in the clean up